coreneuron/io/reports/report_configuration_parser.cpp

```diff
@@ -155,6 +155,7 @@ std::vector<ReportConfiguration> create_report_configurations(const std::string&
             // extra new line: skip
             report_conf.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
         }
+        report.report_index = report.format == "SONATA" ? i : 0;
```

Comment:
```suggestion
        report.report_index =  i;
```

I think we don't need to have this specialised for sonata only?

Reply:
This was causing differences in the binary reports in the CI.

Reply:
Hmm ok! 

How many such reports are different? the new logic of "determinist ordering" of report recording seems cleaner right? In that case, I wonder if we should just change the report files.